libs/database/src/lib/services/content.service.ts

```diff
@@ -1294,4 +1294,115 @@ Message: ${message}`,
       items: result,
     };
   };
+
+  /**
+   *
+   * @param {string} userId
+   * @param {number} maxResults
+   * @param {string} sinceId
+   * @param {string} untilId
+   * @returns
+   */
+  getEngagementFromUser = async (
+    userId: any,
+    maxResults: number = DEFAULT_CONTENT_QUERY_OPTIONS.maxResults,
+    sinceId: string,
+    untilId: string
```

Comment:
```suggestion
    sinceId: string,
    untilId: string,
    maxResults = DEFAULT_CONTENT_QUERY_OPTIONS.maxResults
```
optional value ควรอยู่หลังสุด